src/scripts/main.js

```diff
@@ -21,7 +21,34 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key1 in food) {
+    const ol = document.createElement('ol');
+
+    ol.textContent = key1;
+    document.body.appendChild(ol);
+
+    if (typeof food[key1] === 'object') {
+      for (const key2 in food[key1]) {
+        if (key2 !== undefined) {
+          const li1 = document.createElement('li');
+
+          li1.textContent = key2;
+
+          ol.appendChild(li1);
+
+          for (const key3 in food[key1][key2]) {
```

Comment:
The `for...in` loop should iterate over `data[key1][key2]` instead of `food[key1][key2]` to utilize the `data` parameter.